### DISCUSSION TITLE
part-ii/2016-46/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#275] [Question comments]
Part II 2016 Differential Geometry: Paper 1, Section II, G 

https://questions.tripos.org/part-ii/2016-46/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(a)

(b) Stack of Records theorem.  $X = \mathbb{R}^1$ and $Y = \mathbb{R}^1$,  and let $1 + \frac{1}{n}$ be criticial values.

(c) (not confident) A zero dimensional manifold is a discrete set, since every singleton set is open. The set of critical values is at most 0 dim, since it has measure 0. So suffices to construct $f:X\to Y$ such that the set of critical values has an accumulation point. Consider the bump function $f_n:[0,1]\to\left[0,\frac{1}{n}\right]$, supported on $\left[\frac{1}{n+1},1-\frac{1}{n+1}\right]$ and $=\frac{1}{n^2}$ on $\left[\frac{1}{n},1-\frac{1}{n}\right]$, for $n\geq 3$. Consider the sum $f = f_3+f_5+f_7+\dots$. Then $f\in C^{\infty}$ and the critical values of $f$ forms a sequence with limit $0$. 

</details>

**Reply by ArthurConmy** [reply]
Yeah, works fine. What's not confident?

I was confused as to what a 0D manifold is, though I think it's clear once you know that $\mathbb{ R}^0 = \{ 0 \}$ a single point space, by definitions I can find online.
